packages/trpc/server/routers/viewer/admin/listPaginated.handler.ts

```diff
@@ -46,8 +46,6 @@ const listPaginatedHandler = async ({ input }: GetOptions) => {
     take: limit + 1, // We take +1 as itll be used for the next cursor
     where: {
       ...searchFilters,
-      // To bypass the excludeLockedUsersExtension
-      OR: [{ locked: false }, { locked: true }],
```

Comment:
It is redundant, The searchFilters already includes it

Reply:
How does this fix it?

Reply:
The search filter already has the OR condition, and we are adding another one(same) later, which is causing a conflict now